magda-web-client/src/Components/Dataset/View/DistributionRow.tsx

```diff
@@ -42,16 +42,12 @@ class DistributionRow extends Component<PropType> {
 
     render() {
         const { dataset, distribution } = this.props;
-        let distributionLink;
-        if (!distribution.downloadURL && distribution.accessURL) {
-            distributionLink = distribution.accessURL;
-        } else {
-            distributionLink = `/dataset/${encodeURIComponent(
-                dataset.identifier
-            )}/distribution/${encodeURIComponent(
-                distribution.identifier!
-            )}/?q=${this.props.searchText}`;
-        }
+
+        let distributionLink = `/dataset/${encodeURIComponent(
```

Comment:
Since we don't need to alter the value of `distributionLink ` anymore.
Could we please change it to a `const`?
i.e. 
```
const distributionLink = `/dataset/${encodeURIComponent(
```

Reply:
Done :+1: